app/interactors/backdate/destroy_interactor.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Backdate::DestroyInteractor
+  include Interactor
+
+  delegate :params, :edition, :user, to: :context
+
+  def call
+    Edition.transaction do
+      find_and_lock_edition
+      update_edition
+      update_preview
+    end
+  end
+
+private
+
+  def find_and_lock_edition
+    context.edition = Edition.lock.find_current(document: params[:document])
+
+    unless edition.editable? && edition.first?
+      # FIXME: this shouldn't be an exception but we've not worked out the
+      # right response - maybe bad request or a redirect with flash?
+      raise "Only editable backdated first editions can have their backdated date cleared."
+    end
+  end
+
+  def update_edition
+    updater = Versioning::RevisionUpdater.new(edition.revision, user)
+    updater.assign(backdated_to: nil)
+
+    edition.assign_revision(updater.next_revision, user).save!
```

Comment:
We should check whether the updater has changed and abort if not - not sure if this is covered on the update interactor too

Reply:
It's not in the update interactor.  A quick look through the interactors shows that there are quite a few that don't abort if the updated has changed.  I will add it to this one, and raise a separate PR to add it consistently across the interactors.

Reply:
Thanks. Yeah we should try avoid creating timeline entries when they don't reflect any change to the underlying system state.